tests/suites/test_suite_asn1parse.data

```diff
@@ -514,13 +514,13 @@ traverse_sequence_of:"300705000203123456":0:0:0xff:0x02:"6,0x02,3":0
 Traverse SEQUENCE of INTEGER, skip everything
 traverse_sequence_of:"30080203123456020178":0xff:0x02:0:1:"":0
 
-Traverse SEQUENCE of {NULL, OCTET STRING}, skip NULL: OS, NULL
+Traverse SEQUENCE of NULL, OCTET STRING, skip NULL: OS, NULL
```

Comment:
`{}` are valid characters in test descriptions and they're used in other places.

>  In analyze_outcomes.py, it tries to log the result

That's an escaping bug and I really don't like to have an escaping bug in a script that processes user data. Please undo 929311e9a7c092b54a05d84bc74daa8efdb07422 and fix that bug instead.

This means reviewing the calls to `Results.log` and making sure that the first argument is a template, and any variables are in separate strings. For example:
```
        Results.log("Outcome file (" + outcome_file + ") already exists. " + \
                    "Tests will be skipped.")
```
should be
```
        Results.log("Outcome file {} already exists. " + \
                    "Tests will be skipped.", outcome_file)
```